rmf_site_editor/src/interaction/model.rs

```diff
@@ -72,34 +73,18 @@ pub fn update_model_instance_visual_cues(
         if let Some(tasks) = tasks {
```

Comment:
Technically this isn't hurting anything, but I don't believe any of this block is needed any longer.